presenters/thread.rb

```diff
@@ -76,7 +76,7 @@ def to_hash with_responses=false, resp_skip=0, resp_limit=nil, recursive=true
   #     children, if recursive is true)
   #   response_count
   #     The total number of responses
-  def get_paged_merged_responses(thread_id, responses, skip, limit, recursive=false)
+  def get_paged_merged_responses(thread_id, responses, skip, limit, recursive=true)
```

Comment:
`recursive` was added later as part of a change where we did not want thread to return child comments along with responses. But later on it figured out that default behaviour was to return all responses and child comments. to_hash has the right value for it as default so I have changed it in paginated flow too.